### DISCUSSION TITLE
PiPedal v1.2.41 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#176] [Releases]
PiPedal v1.2.41  includes the following new features:

- Supports Raspberry Pi 5.
- Supports Rasberry Pi OS Bookworm.
- TooB ML allows uploading of models. See below for further details.
- TooBML support for large models (e.g. GuitarML Proteus models)
- Upload .zip file bundles to all File plugin controls.
- Uploads can be organized into sub-directories.
- Various minor improvements to TooB plugin user interfaces.
- PiPedal now monitors LV2 plugin directories for changes, and automatically updates available plugins in the 
  PiPedal UI, without requiring a restart.

You can now upload new models into TooB ML. TooB ML models are compatible with models for the GuitarML Proteus plugin. Currently, the GuitarML Proteus model download pages are the best source for new models to use with TooB ML. To install 
new models in TooB ML, follow these instructions:

- Download Proteus models from https://guitarml.com/tonelibrary/tonelib-pro.html to your local computer or Android
  device. TooB ML supports both Proteus .json models, and also allows you to install model collections contained in .zip files.
- Load the TooB ML plugin.
- click on the Model control.
- click on the Upload button in the lower left corner of the Model selection dialog.
- Upload the .json model file or .zip model collection that you downloaded from The GuitarML Tone Library.

**Comment by 38github** [comment]
I love the vertical file list when using Neural Amp Modeler LV2. The old horisontal view drove me mad at times :) What I also love is that PiPedal's CPU usage is so much lower and stable than MODEP. Now I just hope that there will asap pop up a RPi HAT with ok converters. guitar input with correct impedance match and one or two outputs with a really fair (subjective) price tag.
